### PR TITLE
DX467: Replaced Swagger UI Links with API Ref

### PR DESCRIPTION
**Pull request by JoshBlanks**
Ticket: https://jira.avalara.com/browse/DX-467

Changed most/all (I think I did them all) links from the Swagger UI to the new corresponding API Reference page. I'm linking @ted-spence-avalara on this to make sure my links are all correct :)

Unfortunately looks like eslint/code-editor destroyed a bunch of empty spaces and stuff - so dirty PR, but I only changed links....

### CODE REVIEW

File: _posts/2016-10-04-getting-started-with-avatax-rest-v2.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ So in a moment or two, you should receive an email from support@avalara.com with
...
+As Bob's Artisan Pottery store, we will need to begin by adding a "Company" record to keep track of our sales.  Since Avalara works for all companies both small and large, you can use the AvaTax API whether you have a single corporate entity or whether you manage lots of different registered businesses.  Let's start by listing all the companies defined in this account.In AvaTax, we list companies by calling <a href="https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Companies/">https://sandbox-rest.avatax.com/api/v2/companies</a> - but we need to authenticate against the API.  In order to do this, we need to add a basic authentication header to our web request.  Let's start by creating our basic authentication header.  If you're writing code, your programming language likely supports Base64 encoding directly.  Here's how to construct the basic authentication header in C# - most other programming languages work the same way:
```
Comment (ted-spence-avalara) [inline _posts/2016-10-04-getting-started-with-avatax-rest-v2.md]: This should link directly to "CreateCompany" not to the category "Companies"

Comment (ted-spence-avalara) [inline _posts/2016-10-04-getting-started-with-avatax-rest-v2.md]: https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Companies/CreateCompanies/

File: _posts/2016-10-04-getting-started-with-avatax-rest-v2.md [hunk 2]
```diff
@@ -77,11 +77,11 @@ If you'd like to type along with me without using a programming language, we can
 Of course, to make this work, you'll have to use your username and password instead of Bob McExample's information.  Once you've completed your basic authentication header, let's make an API call to list companies:
 
 <ul class="normal">
-<li>Launch the <b>List Companies</b> API by clicking this URL: <a href="https://sandbox-rest.avatax.com/swagger/ui/index.html#!/Companies/ApiV2CompaniesGet">https://sandbox-rest.avatax.com/swagger/ui/index.html#!/Companies/ApiV2CompaniesGet</a></li>
+<li>Launch the <b>List Companies</b> API by clicking this URL: <a href="https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Companies/">https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Companies/</a></li>
```
Comment (ted-spence-avalara) [inline _posts/2016-10-04-getting-started-with-avatax-rest-v2.md]: https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Companies/QueryCompanies/

File: _posts/2016-10-24-batch-transaction-upload-in-rest-v2.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ For our next step, we need to convert this into a JSON request.
 
 <h2>Building our JSON Request</h2>
 
-In order to upload a batch, you need to know two things: First, the AccountID of your account, and second, the CompanyID of the company you want to upload this transaction for.  The easiest way to do this is to <a href="https://sandbox-rest.avatax.com/swagger/ui/index.html#!/Companies/ApiV2CompaniesGet">list all companies in your account</a>, and copy those values off that result.  Here's how you do it:
+In order to upload a batch, you need to know two things: First, the AccountID of your account, and second, the CompanyID of the company you want to upload this transaction for.  The easiest way to do this is to <a href="https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Companies/">list all companies in your account</a>, and copy those values off that result.  Here's how you do it:
```
Comment (ted-spence-avalara) [inline _posts/2016-10-24-batch-transaction-upload-in-rest-v2.md]: https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Companies/QueryCompanies/

File: _posts/2016-10-24-batch-transaction-upload-in-rest-v2.md [hunk 4]
```diff
@@ -128,7 +128,7 @@ The end result should now look like this:
 Now, our next step is to create this batch file in AvaTax REST v2.  Here's how we do that:
 
 <ul class="normal">
-    <li>Launch the <a href="https://sandbox-rest.avatax.com/swagger/ui/index.html#!/Batches/ApiV2CompaniesByCompanyIdBatchesPost">Batch Creation API</a> in REST v2</li>
+    <li>Launch the <a href="https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Batches/">Batch Creation API</a> in REST v2</li>
```
Comment (ted-spence-avalara) [inline _posts/2016-10-24-batch-transaction-upload-in-rest-v2.md]: https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Batches/CreateBatches/

File: _posts/2016-10-24-batch-transaction-upload-in-rest-v2.md [hunk 5]
```diff
@@ -165,9 +165,9 @@ First, note that the batch you just created is in status "Waiting".  That means
...
+We'll continue by fetching our batch back from the server.  Launch the <a href="https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Batches/">Batch GET API</a> and type in your company ID and the batch ID you got back from your Batch Create API call.  The result you'll get back will either still say <code class="highlight-rouge">"status": "Waiting"</code>, or it will say <code class="highlight-rouge">"Completed"</code> or <code class="highlight-rouge">"Errors"</code>.  If the batch is still <code class="highlight-rouge">"Waiting"</a>, you'll just want to keep checking back periodically to see when it has finished.
```
Comment (ted-spence-avalara) [inline _posts/2016-10-24-batch-transaction-upload-in-rest-v2.md]: https://developer.avalara.com/api-reference/avatax/rest/v2/methods/Batches/GetBatch/

File: _posts/2016-10-04-getting-started-with-avatax-rest-v2.md (hunk 1 above)
Comment (JoshBlanks) [inline _posts/2016-10-04-getting-started-with-avatax-rest-v2.md]: updated

**Comment by ted-spence-avalara** [comment]
Ship it

**Pull request merged**